fml/mapping.cc

```diff
@@ -102,6 +101,47 @@ const uint8_t* NonOwnedMapping::GetMapping() const {
   return data_;
 }
 
+// MallocMapping
+MallocMapping::MallocMapping() : data_(nullptr), size_(0) {}
+
+MallocMapping::MallocMapping(const uint8_t* data, size_t size)
+    : data_(data), size_(size) {}
+
+MallocMapping::MallocMapping(fml::MallocMapping&& mapping)
+    : data_(mapping.data_), size_(mapping.size_) {
+  mapping.data_ = nullptr;
+  mapping.size_ = 0;
+}
+
+MallocMapping::~MallocMapping() {
+  if (data_) {
```

Comment:
This is not necessary as `::free(nullptr)` is safe. In fact, I think clang-tidy even has a fix-it for this sort of deletion.

Reply:
Looked it up, its for `delete` https://clang.llvm.org/extra/clang-tidy/checks/readability-delete-null-pointer.html. Same stuff.

Reply:
nice, done, thanks